src/Symfony/Component/Console/Command/Command.php

```diff
@@ -284,6 +284,10 @@ public function setCode($code)
             throw new \InvalidArgumentException('Invalid callable provided to Command::setCode.');
         }
 
+        if (PHP_VERSION_ID >= 50400 && $code instanceof \Closure) {
```

Comment:
You are right, but what if someone define a user-land `Closure` class ?

IHMO, it's safer to keep the version check (and will be removed in SF3.0 anyway)